burst-gradle-plugin/src/test/projects/androidBuiltInKotlin/build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+import org.jetbrains.kotlin.gradle.tasks.KotlinJvmCompile
+
+buildscript {
+  repositories {
+    maven {
+      url = file("$rootDir/../../../../../build/testMaven").toURI()
+    }
+    mavenCentral()
+    google()
+  }
+  dependencies {
+    classpath("app.cash.burst:burst-gradle-plugin:${project.property("burstVersion")}")
+    classpath("com.android.tools.build:gradle:9.0.0-alpha05")
```

Comment:
not sure how we want to manage this version?

Reply:
TOML sounds good. Would `android -gradlePlugin9` make more sense because this is AGP not KGP?

Reply:
Yep yep you got it